code/modules/status_system/statusEffects.dm

```diff
@@ -2975,3 +2975,32 @@
 		src.art = null
 		qdel(src.glimmer)
 		src.glimmer = null
+
+/datum/statusEffect/art_fissure_corrosion
+	id = "art_fissure_corrosion"
+	effect_quality = STATUS_QUALITY_NEGATIVE
+	var/corrosion_stacks = 0 // 1 stack per tick
```

Comment:
Might be better to have this status cleared on area `Exited()`, and track state via atomproperty, rather than keeping an infinite duration status ticking forever on any obj that's touched the floor was and then removed